docs/help.rst

```diff
@@ -45,50 +45,69 @@ to ask for it. We will need to ask for help from our colleagues, from our superv
 
 We are a small team of researchers and developers, and we are not experts in *everything at once*. Each one of us has
 a specific expertise, and we are trying to use this expertise to create Junifer. When we thought of Junifer, we thought
-about researcher's problems, and try to come up with the best way to help them, by building a tool that is easy to use,
-that is easy to learn, and that is easy to understand. Most importantly, we are here to help. We are here to help you
+about researcher's problems, and try to come up with the best way to help them, by building a tool that is easy to 
+understand, learn and use. Most importantly, we are here to help. We are here to help you
 and your research.
 
-If you have any questions, if you have any problems, if you have any suggestions, please do not hesitate to contact
-us. We will be happy to help you, and we will be happy to hear from you.
+If you have any questions, problems or suggestions, please do not hesitate to contact us. We will be happy to help you
+ and we will be happy to hear from you.
 
-Seems nice, no? But we have one condition: **helps us help you**.
+Seems nice, no? But we have one condition: **help us help you**.
 
-We are not mind readers. We cannot know what you are thinking, and we cannot know what you are trying to do. Unless you
+Communication is the key for you to help us and in turn help you solve your problems. We are not mind readers. We
```

Comment:
I would not put the "mind readers" part specifically and words in and around it.